content/articles/securing-mongodb-in-linux/index.md

```diff
@@ -0,0 +1,229 @@
+
+### Securing MongoDB in Linux
+
+### Content
+
+- [Prerequisites](#prerequisites)
+- [Introduction](#introduction)
+- [Check the environment](#check-the-environment)
+- [Securing MongoDB](#securing-mongodb)
+- [Create a user with system access](#create-a-user-with-system-access)
+- [Turn on authentication](#turn-on-authentication)
+- [Authentication Testing](#authentication-testing)
+- [Wrapping it up](#wrapping-it-up)
+
+### prerequisites

```

Comment:
Ensure you always capitalize the first letter in your heading. And only the first word in the heading needs to be capitalized. Please got through all the headings and edit accordingly